documentation/modules/ROOT/pages/01-pattern.adoc

```diff
@@ -1,7 +1,28 @@
 == The story behind this solution pattern
 
-A description of the story that was used to build the demo and architectures of this SP.
+Cabs is a fictional transportation company.
+Engineers at Cabs were struggling with their transactional-style, monolith application.
+They recently started an effort to modernize it.
+
+The team saw a https://www.youtube.com/watch?v=Rc5IO6S6ZOk[talk on _Event Mesh_].
+The talk presented the advantages of moving beyond transactional architectures in favor of eventual consistency.
+By leveraging event meshes with technologies like Knative, developers can achieve decoupled, reliable microservices without extensive re-engineering.
+This solution addresses inefficiencies and aligns distributed systems with real-world business processes.
+The team had figured out they could leverage the _CQRS_ pattern together with _Knative's Event Mesh_
+to modernize their application in a non-extrusive way.
 
 == The Solution
 
-This is a summary of the solution
+The core of this solution is an event mesh—a dynamic, flexible layer that routes, retries, and processes asynchronous events across distributed components.
```

Comment:
I'd really state that it (the event mesh) is an infrastructure piece

Reply:
Hmm, what your proposal. Maybe just write "flexible infrastructure layer" instead of just "flexible layer"?